doc/develop/extending.rst

```diff
@@ -330,6 +330,46 @@ To read an acquisition file you can use code such as:
     print das[0].metadata
 
 
+Starting odemis from the terminal/console
+=================================================
+
+After setting up the development environment it is possible to start odemis via the terminal.
+It is also possible to specify a specific configuration (*.yaml) file used for staring odemis.
+
+
+Starting odemis
+-----------------------
+
+Odemis can be started from the terminal by typing the following command in the terminal
+(from ``~/development/odemis``)::
+
+    odemis-start
+
+The default configuration file (*.yaml) is used, which can be found and changed in
```

Comment:
`The default microscope file is defined in the configuration file, we can be found and changed in /etc/odemis.conf`.